CONTRIBUTING.md

```diff
@@ -73,6 +73,27 @@ If you have questions about the project, using PyScript, or anything else, pleas
 
 Now that node and npm have both been updated `make setup` should work, and you can continue [setting up your local environment](#setting-up-your-local-environment) without problems (hopefully).
 
+## Creating a Pull Request
+
+After you fork PyScript make sure to add a new remote. You can do that using the command:
+
+```
+git remote add remote_nickname remote_url
+```
+
+You should have one remote that corresponds to your local repository and one remote that corresponds the upstream PyScript repository.
+
+We recommend you to create a new local branch and develop your changes in this brance=h. You can do that by using:
+
+```
+git checkout -b branch_name #creates new branch
+git checkout branch_name #goes to branch
+```
+
+The changes should try to be kept within the scope of the issue, or within a well defined scope that should be clearly explained on the PR title. If you can't think of a title that explains your PR well might be a sign that you need to split it in two or more PRs!
```

Comment:
Think this sentence could be rephrased to make it more readable. Something like "Please try to keep the changes on your PR within the scope of the issue,..."

Reply:
Also, you could add a `'` between "your PR" and "well"